crates/matrix-sdk-ui/src/timeline/threaded_events_loader.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright 2024 The Matrix.org Foundation C.I.C.
```

Comment:
correct dates? where we're going, we don't need dates!

```suggestion
// Copyright 2025 The Matrix.org Foundation C.I.C.
```